src/IssueViz/Editor/SelectedIssueTagging/Adornment/IssueLocationAdornment.cs

```diff
@@ -60,8 +58,15 @@ public IssueLocationAdornment(IAnalysisIssueLocationVisualization locationViz, I
 
         public void Update(IFormattedLineSource formattedLineSource)
         {
-            FontSize = formattedLineSource.DefaultTextProperties.FontRenderingEmSize;
-            FontFamily = formattedLineSource.DefaultTextProperties.Typeface.FontFamily;
+            var textBlock = Child as TextBlock;
+
+            textBlock.Foreground = formattedLineSource.DefaultTextProperties.ForegroundBrush;
+            textBlock.FontSize = formattedLineSource.DefaultTextProperties.FontRenderingEmSize - 2;
```

Comment:
I wanted to make the number stand out without making the adornment too big, so I made the font Bold (line 47) and the font a bit smaller as a compromise. I do the same thing in the Viz panel.